collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_113.xml

```diff
@@ -41,46 +44,158 @@
+                        <rubric>Incipit prefatio libri huius qui est prima pars collationum</rubric>
+                        <incipit> Debitum quod beatissimo papę Castori in eorum voluminum prefatione promissum est</incipit>
+                        <explicit>collationes eorum &amp; instituta properemus</explicit></msItem>
+                     <msItem>
+                        <locus>(fol. 3v)</locus>
+                        <note>Capitula list</note>
+                        <rubric>Incipiunt capitula collationis prime partis</rubric>
+                        <incipit>.i. De habitacione Scythii &amp; proposito abbatis Moysi </incipit>
+                        <explicit>xix De quadripartita discretionis ratione. De sermone doctoris secundum audientius merita</explicit>
+                        <finalRubric>Expliciunt capitula</finalRubric>
+                        <note>Numbered i–xix, but two capitula appear after each of ‘xi’, ‘xiii’, ‘xiiii’, and ‘xix’; in the text itself the numbering reaches ‘xxiii’, as normal.</note></msItem>
+                     <msItem>
+                        <locus>(fols. 3v–109v)</locus>
+                        <rubric> Incipiunt sanctorum patrum decem collationes a venerabili Cassiano insigniter prolate. Collatio prima abbatis Moysi de monachi destinatione vel fine. De habitatione Scythii &amp; proposito abbatis Moysi</rubric>
+                        <incipit>Dum in heremo Scythii ubi monachorum probatissimi patres &amp; omnes commorabatur perfectio </incipit>
+                        <explicit>versiculi huius meditatione servaverint</explicit>
+                        <finalRubric>Explicit liber collationum</finalRubric>
+                        <note>Capitula lists precede each of the ten books, which start on fols. 4r, 15r, 25, 35r, 43r, 54r, 64r, 77r, 88v, and 95r.</note>
+                        <note>Fol. 75 is a replacement leaf, 12th century.</note>
+                        <note>A bifolium in quire 10 (fols. 86 and 89) is bound out of place: fol. 86 should follow fol. 83, and fol. 89 should follow fol. 91. The lower right corner of fol. 83v has a note ‘causas atque insidias per saltu duo folia et inveniens cum tali signum î’, and the upper left corner of fol. 86r, which begins with the words ‘causas atque insidias’, has the same sign. All except the last leaf of the following quire is missing, from ‘... orationem et obsecrationem ac postulal- ||’ to ‘|| iugitate cuncta quę secundum deum ...’ (edn. p. 260 l. 21 to p. 279 l. 10).</note>
+                        </msItem>
+                     <msItem>
+                        <note>Following the end of the text are two added pairs of verses in small 13th-century script:</note>
+                        <msItem>
+                           <note><q>Equor erit siccus cum pauper habebit amicum / Hoc si contigerit bos citharedus erit</q> (<bibl type="repertory">Walther, <title>Proverbia</title>, no. 625</bibl>)</note></msItem>
```

Comment:
It would be nice to use `<l>` tags in lines of verse and have them render properly, though I don't think this actually works in the catalogue at the moment.